.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "dropwizard-swagger"]
```

Comment:
Shouldn't be there - thought I had taken it out. I'll need to check it when I get home.

Sent from my iPhone

> On Apr 14, 2015, at 10:34 PM, Carlton Whitehead <notifications@github.com> wrote:
> 
> In .gitmodules:
> 
> > @@ -0,0 +1,3 @@
> > +[submodule "dropwizard-swagger"]
> Why did you add this as a submodule? The dependency declaration in the pom.xml should suffice
> 
> —
> Reply to this email directly or view it on GitHub.
>